### DISCUSSION TITLE
当你感觉负重前行时，一定有人在替你岁月静好

### DISCUSSION THREAD

**Discussion by Ningmeng-yp** [WeNeedHome/SummaryOfLoanSuspension#968] [General]
唐山打人事件，郑州710，业主集体停贷维权，到现在没有一点声，石沉大海，没有人敢为他们发声，让人心凉。如果事实敢于公诸于世，又何必藏着掖着。
花了于建造成本数十倍的价买的房子，只求能稳妥交房，现在连这也成为了奢望。地产商摆烂了，贷款机构和监管方只会出来扯皮，没人愿意解决问题，只会告诉业主，不还钱就上征信，冻结你的账号，查封你的车子，上黑名单，坐不了飞机高铁，禁止你的孩子考公，影响你的人生。
那么房子烂尾中违反监管、违反法律的人的责任呢，为什么没有人去追究，反而来约束自始至终没有错的普通人呢？
他只是想要回房子，他有什么错？

**Comment by imhappyok** [comment]
错在生于平民家，没办法，你还想掀桌子吗？

**Comment by BH-Parity** [comment]
我们现在没有多少底牌，唯一的底牌就是不要忘记，持续表达诉求，这也是一种压力，会产生影响的。

**Comment by wedxz** [comment]
一个烂尾楼的业主居然跟我说为什么要把统计的这些数据放到这里来，这不是给境外势力递刀子吗？
真是一群贱民，活该捶他们

**Reply by TALK1949** [reply]
不要小看这么多年墙内的舆论引导，他们跟那些跑到这里来不让别人谈政治没差太多，等他们成为他们口中的境外势力，他们才会明白他们口中的境外势力到底是啥。

**Comment by wedxz** [comment]
> 唐山打人事件，郑州710，业主集体停贷维权，到现在没有一点声，石沉大海，没有人敢为他们发声，让人心凉。如果事实敢于公诸于世，又何必藏着掖着。
> 花了于建造成本数十倍的价买的房子，只求能稳妥交房，现在连这也成为了奢望。地产商摆烂了，贷款机构和监管方只会出来扯皮，没人愿意解决问题，只会告诉业主，不还钱就上征信，冻结你的账号，查封你的车子，上黑名单，坐不了飞机高铁，禁止你的孩子考公，影响你的人生。
> 那么房子烂尾中违反监管、违反法律的人的责任呢，为什么没有人去追究，反而来约束自始至终没有错的普通人呢？
> 他只是想要回房子，他有什么错？

错在打搅了领导的春秋大梦
